package.json

```diff
@@ -39,7 +39,8 @@
     "react-toast-notifications": "^2.4.0",
     "set-interval-async": "1.0.33",
     "showdown": "^1.9.1",
-    "tesseract.js": "^4.0.2"
+    "tesseract.js": "^4.0.2",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.1/xlsx-0.20.1.tgz"
```

Comment:
looks like sheetjs is not publishing to [npm registry](https://www.npmjs.com/package/xlsx) anymore https://git.sheetjs.com/sheetjs/sheetjs/issues/3055, i'm not sure if our vulnerability scan tools support libraries like this

Reply:
Scanners scan built artifacts, and they identify some CVEs by examining package.json and subdeps. I have never heard of an npm requirement.

Reply:
@joshuali925 @wbeckler 

I have updated the pull request with transition from `xlsx` to `ExcelJS`

This change was motivated by several factors:
* Robust API
* Advanced styling options available out of the box
* Availability on npm

Reply:
thanks. @sumukhswamy could you help to take a look at this when you can?